stack.yaml

```diff
@@ -1,8 +1,14 @@
-resolver: lts-14.1
+resolver: nightly-2019-09-29
 
 extra-deps:
-  - parser-combinators-1.2.0
+  - tasty-silver-3.1.13
+  - regex-tdfa-1.2.3.2  # for tasty-silver
   - toml-parser-0.1.0.0
 
   - htoml-megaparsec-2.1.0.3
   - composition-prelude-2.0.2.1  # needed for htoml-megaparsec
+
+allow-newer: true
+
+ghc-options:
+  "$locals": -Wmissing-deriving-strategies
```

Comment:
Is this needed if we have it in common stanza? Or `stack` doesn't recognize it?

Reply:
:thinking:, good question, I'm actually not sure about this. But I guess you're right, I'll remove it.